File: dart-jaguar/lib/model/collections_collection.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'collections_collection.jser.dart';

class CollectionsCollection {
  
  final String cid;
  
  final String createdAt;
  
  final String description;
  
  final String name;
  
  final int userId;
  
  final String uuid;
  

  CollectionsCollection(
    

{
     this.cid = null,  
     this.createdAt = null,  
     this.description = null,  
     this.name = null,  
     this.userId = null,  
     this.uuid = null 
    
    }
  );

  @override
  String toString() {
    return 'CollectionsCollection[cid=$cid, createdAt=$createdAt, description=$description, name=$name, userId=$userId, uuid=$uuid, ]';
  }
}

@GenSerializer()
class CollectionsCollectionSerializer extends Serializer<CollectionsCollection> with _$CollectionsCollectionSerializer {

}

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Collections.Collection.scala
package io.swagger.server.model


/**
 * @param cid 
 * @param createdAt 
 * @param description 
 * @param name 
 * @param userId 
 * @param uuid 
 */
case class Collections.Collection (
  cid: Option[String],
  createdAt: Option[String],
  description: Option[String],
  name: Option[String],
  userId: Option[Int],
  uuid: Option[String]
)

